ofl/notosansnandinagari/METADATA.pb

```diff
@@ -0,0 +1,22 @@
+name: "Noto Sans Nandinagari"
+designer: "Google"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2022-12-12"
+fonts {
+  name: "Noto Sans Nandinagari"
+  style: "normal"
+  weight: 400
+  filename: "NotoSansNandinagari-Regular.ttf"
+  post_script_name: "NotoSansNandinagari-Regular"
+  full_name: "Noto Sans Nandinagari Regular"
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/nandinagari)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+source {
+  repository_url: "https://github.com/notofonts/nandinagari.git"
+  archive_url: "https://github.com/notofonts/nandinagari/releases/download/NotoSansNandinagari-v1.001/NotoSansNandinagari-v1.001.zip"
+}
+is_noto: true
```

Comment:
If a font is missing subset and language, it means the packager did not assign one because it could not find anything suitable in the glyphsets repo or in the lang repo. So we need fixes in those repos before merging this.